README.md

```diff
@@ -272,6 +347,31 @@ For cluster to get properly authorized it is expected that firewall is already c
       roles:
          - { role: 'ondrejhome.ha-cluster-pacemaker', cluster_name: 'vmware-cluster', cluster_configure_fence_xvm: false, cluster_configure_fence_vmware_rest: true, cluster_configure_stonith_style: 'one-device-per-cluster' }
 
+**Example playbook Resources configuration** .
+
+    - hosts: cluster
+      vars:
+        cluster_property:
+          - name: 'stonith-enabled'
+            value: 'false'
+          - name: 'no-quorum-policy'
+            value: 'ignore'
+        cluster_resource:
+          - name: apache2
+            resource_type: 'systemd:apache2'
+            options: 'meta migration-threshold=2 op monitor interval=20s timeout=10s'
+          - name: cluster_vip
+            resource_type: 'ocf:heartbeat:IPaddr2'
+            options: 'ip=192.168.1.150 cidr_netmask=24 meta migration-threshold=2 op monitor interval=20'
+        cluster_constraint_colocation:
+          - resource1: 'cluster_vip'
+            resource2: 'apache2'
+            score: 'INFINITY'
+        cluster_resource_defaults:
+          - name: 'failure-timeout'
+            value: '30'
+      roles:
+         - { role: 'ondrejhome.ha-cluster-pacemaker', cluster_name: 'apache-cluster', cluster_configure_fence_xvm: false }
```

Comment:
Would it be possible to remove `cluster_configure_fence_xvm: false` from here as well?

Reason is that it is not necessary for allowing the cluster resource configuration nor cluster property changes. My preference is on having rather shorter examples with only needed options to describe the new situations.

Reply:
Yes sure! it should changed now

Reply:
ok, looks good